h/services/annotation_json_presentation.py

```diff
@@ -48,16 +50,15 @@ def eager_load_documents(query):
                 for ann in annotations]
 
     def _get_presenter(self, annotation_resource):
-        return presenters.AnnotationJSONPresenter(annotation_resource,
-                                                  self.formatters)
+        if self.json_flavor == 'jsonld':
+            return presenters.AnnotationJSONLDPresenter(annotation_resource)
```

Comment:
This branch needs to be tested, we do that for the normal JSON presenter [here](https://github.com/hypothesis/h/blob/2f28db72cf8cb6764af6ddda067c1d1cd8b06751/tests/h/services/annotation_json_presentation_test.py#L44-L47) and [here](https://github.com/hypothesis/h/blob/2f28db72cf8cb6764af6ddda067c1d1cd8b06751/tests/h/services/annotation_json_presentation_test.py#L57-L62).